### ISSUE TITLE
linter-pylint wont work after update to 2.0.0

### ISSUE THREAD

**Issue by setaou** [AtomLinter/linter-pylint#206]
After the update to 2.0.0, the linter wont work anymore. Pylint seems to run (according to the background process history) but nothing is displayed in Atom.

The following error appears in the console every time the linter is run:

```
Uncaught (in promise) TypeError: Cannot read property 'start' of undefined
    at messageKey (/home/herve/.atom/packages/linter/lib/helpers.js:91:68)
    at Object.normalizeMessages (/home/herve/.atom/packages/linter/lib/helpers.js:120:19)
    at promises.push.Promise.then._this2.emitter.emit.number (/home/herve/.atom/packages/linter/lib/linter-registry.js:134:19)
```

**Comment by Arcanemagus** [comment]
How are you triggering this? I'm not running into it on my machine, and the CI builds are passing...

**Comment by Arcanemagus** [comment]
Found the bug, but now I have no idea how it was working at all 😕.

**Comment by Arcanemagus** [comment]
Fix published in v2.0.1, sorry about that!
